custom-metrics-stackdriver-adapter/pkg/adapter/translator/utils/filter_builder.go

```diff
@@ -62,13 +74,15 @@ var (
 		cluster:      "resource.labels.cluster",
 		location:     "resource.labels.location",
 		namespace:    "resource.labels.namespace",
+		nodes:        "resource.labels.instance",
```

Comment:
Do we also want to update it here?

Reply:
I did some testing, I think you are right, "metric.labels.node" is the correct way to get the node name: 
- if we filter by "resource.labels.instance", then we will get timeseries that contains both value with "node" label and value without "node" label, which doubles the result.
- if we filter by "metric.labels.node", then we will only the the value with "node" label.
i will update all the use cases, thanks for pointing it out